sanitizer/_text.js

```diff
@@ -1,29 +1,20 @@
-var check = require('check-types'),
-  text_analyzer = require('pelias-text-analyzer');
+const check = require('check-types');
+const _ = require('lodash');
 
 // validate texts, convert types and apply defaults
 function sanitize( raw, clean ){
-
   // error & warning messages
-  var messages = { errors: [], warnings: [] };
+  const messages = { errors: [], warnings: [] };
```

Comment:
`const` creates immutable binding. For primitive values (numbers, strings, etc), the value cannot change. But for an object, the values can still change. 

```javascript
const a = { };
a = 7; // throws TypeError: Assignment to constant variable.
a.b = 4; // no error
a.b = 5; // no error
```